client/components/site-selector/index.jsx

```diff
@@ -212,6 +212,10 @@ export default React.createClass( {
 		const recentSites = sites.map( function( site ) {
 			var siteHref;
 
+			if ( ! site ) {
```

Comment:
Oh I see, the null is the ReactElement to render and we're returning undefined later on anyway so this fits. Looks good.